suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -794,17 +555,13 @@ def get_email_attachment(filename: str, filetype: str, data: Any) -> MIMEBase:
         data: The attachment data
 
     """
-    maintype: str
-    subtype: str
-    payload: str
-    if filetype == "xlsx":
-        maintype = "application"
-        subtype = "vnd.ms-excel"
-        payload = data.getvalue()
-    elif filetype == "csv":
+    maintype = "application"
+    subtype = "vnd.ms-excel"
+    payload: str | bytes = str(data)
```

Comment:
We need to keep using `getvalue()`:
```suggestion
    payload: str | bytes = data.getvalue()
```